### PR TITLE
ci: migrate to orb tools 11

### PR DESCRIPTION
**Pull request by HennaAbbas**
migration from  to orb tools 11

### CODE REVIEW

File: .circleci/test-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+version: 2.1
+orbs:
+  windows: circleci/windows@dev:<<pipeline.git.revision>>
+  orb-tools: circleci/orb-tools@11.1
+filters: &filters
+  tags:
+    only: /.*/
+jobs:
+  test-server-2019:
```
Comment (KyleTryon) [inline .circleci/test-deploy.yml:9]: Ensure each job that has been imported by the migrate script here is tested in the workflow below. I'll tag it below.

File: .circleci/test-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          filters: *filters
+      - orb-tools/pack:
+          filters: *filters
+      - orb-tools/publish:
+          orb-name: circleci/windows
+          vcs-type: << pipeline.project.type >>
+          pub-type: production
+          requires:
+            - orb-tools/pack
+            - command-tests
```
Comment (KyleTryon) [inline .circleci/test-deploy.yml:59]: Remove this as well.

File: .circleci/test-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      # Make sure to include "filters: *filters" in every test job you want to run as part of your deployment.
+      - command-tests:
+          filters: *filters
+      - orb-tools/pack:
+          filters: *filters
+      - orb-tools/publish:
+          orb-name: circleci/windows
+          vcs-type: << pipeline.project.type >>
+          pub-type: production
+          requires:
```
Comment (KyleTryon) [inline .circleci/test-deploy.yml:57]: Ensure every job that was imported above is tested here. Take a look at the original workflow (from the current source, before the migrate script was ran) to see how those jobs were ran previously. This publish job should not run until each of the test jobs above has ran first.

**Review by KyleTryon — CHANGES_REQUESTED** [review]
I modified the title of the PR, the `[semver:segment]` syntax was proprietary to CircleCI/Orbs and no longer needed. We now use https://www.conventionalcommits.org/en/v1.0.0/

**Comment by felicianotech** [comment]
@HennaAbbas Still working on this?

File: .circleci/test-deploy.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      size: medium
+      version: << parameters.version >>
+      variant: << parameters.variant >>
+    steps:
+      - run: Write-Host 'Hello, Windows'
+workflows:
+  test-deploy:
+    jobs:
+      # Make sure to include "filters: *filters" in every test job you want to run as part of your deployment.
+      - command-tests:
```
[the remark this answers is not in the document]
Comment (HennaAbbas) [reply .circleci/test-deploy.yml]: removed line 49

File: .circleci/test-deploy.yml (hunk 1 above)
Comment (HennaAbbas) [reply .circleci/test-deploy.yml:9]: ran test-server-2019,  test-server-2019-cude,  test-server-2022

**Pull request closed without merging**